### ISSUE TITLE
Clarify: “ violate the rules of hooks”

### ISSUE THREAD

**Issue by derolf** [SvelteStack/svelte-query#57]
https://sveltequery.vercel.app/guides/parallel-queries

> Dynamic Parallel Queries with useQueries
> 
> If the number of queries you need to execute is changing from render to render, you cannot use manual querying since that would violate the rules of hooks. Instead, Svelte Query provides a useQueries hook, which you can use to dynamically execute as many queries in parallel as you'd like.

I am verify familiar with the concept of hooks and all their pitfalls in React, but this is the first time I read something mentioning rules of hooks in Svelte.

Can you please elaborate what can go one calls useQuery conditionally?

**Comment by paolotiu** [comment]
I think the docs were basically copied and pasted from react-query.

**Comment by derolf** [comment]
Yeah, that’s what I figured out as well. Anyway, I think the docs should be fixed…

**Comment by SomaticIT** [comment]
Since `useQuery` creates a `Readable` store. It can't be conditionnally executed (svelte compiler will try to subscribe to it anyway).
